raiden/network/transport/matrix/transport.py

```diff
@@ -932,9 +932,9 @@ def _handle_sync_messages(self, sync_messages: MatrixSyncMessages) -> bool:
         call_messages: List[ReceivedCallMessage] = list()
 
         for room, room_messages in sync_messages:
-            raiden_messages, call_messages = self._validate_matrix_messages(room, room_messages)
-            raiden_messages.extend(raiden_messages)
-            call_messages.extend(call_messages)
+            _raiden_messages, _call_messages = self._validate_matrix_messages(room, room_messages)
+            raiden_messages.extend(_raiden_messages)
+            call_messages.extend(_call_messages)
```

Comment:
🤯
Very nice catch. That this worked at all is quite surprising.

I'd even say the names should be more different. 
Maybe `next_raiden_messages` and `next_call_messages`.

Reply:
> Very nice catch. That this worked at all is quite surprising.

Yeah, this was causing issues with WebRTC signaling and Fred caught while we were debugging.
I wonder how anything worked at all.

> I'd even say the names should be more different.
> Maybe next_raiden_messages and next_call_messages.

:+1: